app/Console/Components/Npm/FetchTotals.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace App\Console\Components\Npm;
+
+use App\Events\Packagist\TotalsFetched;
+use Developmint\NpmStats\NpmStats;
+use GuzzleHttp\Client;
+use Illuminate\Console\Command;
+
+class FetchTotals extends Command
+{
+    protected $signature = 'dashboard:fetch-npm-totals';
+
+    protected $description = 'Fetch totals for all our NPM packages';
+
+    public function handle()
+    {
+        $npmStats = new NpmStats(new Client());
+
+        $packageList = collect([
+            "spatie-scss",
+            "form-backend-validation",
+            "vue-save-state",
+            "@spatie/blender-js",
+            "npm-install-peers",
+            "eslint-config-spatie",
+            "vue-tabs-component",
+            "@spatie/blender-css",
+            "vue-expose-inject",
+            "@spatie/blender-media",
+            "vue-table-component",
+            "@spatie/blender-content-blocks",
+            "font-awesome-filetypes",
+            "@spatie/attachment-uploader",
+            "postcss-assign",
+            "@spatie/scss",
+            "spatie-dom"
+        ]);
+
+        /*
+         * First idea was to filter the non-scoped and scoped packages so at least the
+         * scoped packages could've been processed by bulk. This would work, but the
+         * allowed range for bulk is lower than normal, which would lead to wrong,
+         * incomplete or weird data in some cases
+         *
+         */
+        list($scoped, $nonScoped) = $packageList->partition(function ($package) {
```

Comment:
use PHP 7.1 array destructuring here.

Reply:
I ditched the bulk operation logic as you recommended, so these lines aren't a thing anymore :)